packages/admin-next/dashboard/src/lib/api-v2/store.ts

```diff
@@ -8,7 +8,7 @@ export const useV2Store = ({ initialData }: { initialData?: any }) => {
     { initialData }
   )
 
-  const store = data.stores[0]
+  const store = data?.stores[0]
```

Comment:
nit: not really related to your PR, but make sense to manually set the error in the case where isLoading is false and the stores array is empty, to have the same effect as if the current useAdminStore endpoint fails. Otherwise, you might end up with a false positive, where loading is completed and no error is thrown, but at the same time store is undefined, causing the admin to break.